app/src/main/res/values-es/strings.xml

```diff
@@ -77,11 +79,16 @@
         <item quantity="one">día %d</item>
         <item quantity="other">%d días</item>
     </plurals>
+    <string name="textFieldOptional">(Opcional)</string>
+    <string name="transferMessagePlaceholder">¿Algún mensaje que quiera transmitir?</string>
+    <string name="transferRecipientAddressPlaceholder">Envíe los archivos a</string>
     <string name="transferSendButton">Enviar</string>
+    <string name="transferSenderAddressPlaceholder">Su dirección de correo electrónico</string>
     <plurals name="transferSpaceLeft">
         <item quantity="one">Queda %s</item>
         <item quantity="other">Quedan %s</item>
     </plurals>
+    <string name="transferTitlePlaceholder">Dé un título a su transferencia</string>
```

Comment:
The verbs are conjugated with the polite form instead of the second person singular form